prometheus_client/openmetrics/exposition.py

```diff
@@ -67,24 +67,87 @@ def generate_latest(registry):
                             floatToGoString(s.exemplar.value),
                         )
                 else:
-                    exemplarstr = ''
+                    exemplarstr = ''   
+                        
                 timestamp = ''
                 if s.timestamp is not None:
                     timestamp = f' {s.timestamp}'
+                
+                native_histogram = ''
+                positive_spans = ''
+                positive_deltas = ''
+                negative_spans = ''
+                negative_deltas = ''
+                pos = False
+                neg = False
+                     
+                if s.native_histogram:                
+                    # Initialize basic nh template
+                    nh_sample_template = '{{count:{},sum:{},schema:{},zero_threshold:{},zero_count:{}'
+
+                    args = [
+                        s.native_histogram.count_value,
+                        s.native_histogram.sum_value,
+                        s.native_histogram.schema,
+                        s.native_histogram.zero_threshold,
+                        s.native_histogram.zero_count,
+                    ]
+
+                    # Signal presence for pos/neg spans/deltas
+                    pos = False
+                    neg = False
```

Comment:
nit: these just shadow `s.native_histogram.pos_spans` and `s.native_histogram.neg_spans` so I'd just write that, but no strong opinion